jsonschema/apis/City_v1_000.json

```diff
@@ -0,0 +1,312 @@
+        } ],
+        "responses" : {
+          "200" : {
+            "description" : "Operação realizada com sucesso!",
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                  "$ref" : "https://raw.githubusercontent.com/totvs/ttalk-standard-message/cities/v1/2_000/jsonschema/schemas/City_2_000.json#/definitions/PagedCities"
+                }
+              }
+            }
+          },
+          "400" : {
+            "description" : "Erro na requisição!",
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                  "$ref" : "https://raw.githubusercontent.com/totvs/ttalk-standard-message/master/jsonschema/apis/types/totvsApiTypesBase.json#/definitions/ErrorModel"
+                }
+              }
+            }
+          }
+        }
+      },
+      "post" : {
+        "tags" : [ "city" ],
+        "summary" : "Inclui uma nova cidade",
+        "description" : "Inclui uma cidade conforme dados enviados na requisição.",
+        "operationId" : "postCity",
+        "x-totvs" : {
+          "productInformation" : [ {
+            "product" : "logix",
+            "available" : true,
+            "note" : "Este verbo está disponível com o parâmetro Authorization.",
+            "minimalVersion" : "12.1.23"
+          } ]
+        },
+        "parameters" : [ {
+         "$ref" : "https://raw.githubusercontent.com/totvs/ttalk-standard-message/master/jsonschema/apis/types/totvsApiTypesBase.json#/parameters/Authorization"
+        }],
+        "requestBody" : {
+          "description" : "Cidade a ser incluída.",
+          "content" : {
+            "application/json" : {
+              "schema" : {
+                "$ref" : "https://raw.githubusercontent.com/totvs/ttalk-standard-message/cities/v1/2_000/jsonschema/schemas/City_2_000.json#/definitions/CityType"
+              }
+            }
+          }
+        },
+        "responses" : {
+          "201" : {
+            "description" : "Operação realizada com sucesso!",
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                  "$ref" : "https://raw.githubusercontent.com/totvs/ttalk-standard-message/cities/v1/2_000/jsonschema/schemas/City_2_000.json#/definitions/CityType"
+                }
+              }
+            }
+          },
+          "400" : {
+            "description" : "Erro durante a inclusão da cidade!",
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                  "$ref" : "https://raw.githubusercontent.com/totvs/ttalk-standard-message/master/jsonschema/apis/types/totvsApiTypesBase.json#/definitions/ErrorModel"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/city/{cityId}" : {
```

Comment:
Erro no Travis: 1) OpenAPI - City_v1_000.json  - Schemas:  should contain the same Id property name in URL and body
Isso quer dizer que você precisa ter a propriedade "cityId" no corpo do seu JSON. Recomendaria você trocar aqui na URL para {CityId}  (com o 'C' maiúsculo) e adicionar essa propriedade no corpo do JSON (schemas de resposta).

Reply:
Bom dia Francisco,
Realizei a troca da propriedade "cityId" da URL pela propriedade "InternalId" que já existe no schema da mensagem.